incubator/graylog/templates/statefulset.yaml

```diff
@@ -0,0 +1,247 @@
+                fi
+              {{- end }}
+                # Start Graylog
+                echo "Starting graylog"
+                # Original docker-entrypoint.sh in Graylog Docker will error while executing since you can't chown readonly files in `config`
+                # exec /docker-entrypoint.sh graylog
+                echo "Graylog Home ${GRAYLOG_HOME}"
+                echo "Graylog User ${GRAYLOG_USER}"
+                echo "JVM Options ${GRAYLOG_SERVER_JAVA_OPTS}"
+                gosu ${GRAYLOG_USER} "${JAVA_HOME}/bin/java" ${GRAYLOG_SERVER_JAVA_OPTS} \
+                    -jar \
+                    -Dlog4j.configurationFile=${GRAYLOG_HOME}/config/log4j2.xml \
+                    -Djava.library.path=${GRAYLOG_HOME}/lib/sigar/ \
+                    -Dgraylog2.installation_source=docker \
+                    ${GRAYLOG_HOME}/graylog.jar \
+                    server \
+                    -f ${GRAYLOG_HOME}/config/graylog.conf ${GRAYLOG_SERVER_OPTS}
+          env:
+            - name: GRAYLOG_SERVER_JAVA_OPTS
+              {{- $javaOpts := "-Djava.net.preferIPv4Stack=true -XX:NewRatio=1 -server -XX:+ResizeTLAB -XX:+UseConcMarkSweepGC -XX:+CMSConcurrentMTEnabled -XX:+CMSClassUnloadingEnabled -XX:+UseParNewGC -XX:-OmitStackTraceInFastThrow" }}
+              {{- if .Values.graylog.heapSize -}}
+              value: "{{ $javaOpts }} {{ printf "-Xms%s -Xmx%s" .Values.graylog.heapSize .Values.graylog.heapSize}}"
+              {{- else if .Values.graylog.resources.requests.memory -}}
+              {{ $heapSize := regexFind "[\\d]+" .Values.graylog.resources.requests.memory}}
+              value: "{{ $javaOpts }} {{ printf "-Xms%sm -Xmx%sm" $heapSize $heapSize }}"
+              {{- else -}}
+              value: "{{ $javaOpts }}"
+              {{- end }}
+            - name: GRAYLOG_PASSWORD_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "graylog.fullname" . }}
+                  key: graylog-password-secret
+            - name: GRAYLOG_ROOT_PASSWORD_SHA2
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "graylog.fullname" . }}
+                  key: graylog-password-sha2
+          ports:
+            - containerPort: 9000
+              name: graylog
+          {{- with .Values.graylog.input }}
+          {{- if .udp }}
+          {{- range .udp.ports }}
+            - containerPort: {{ .port }}
+              name: {{ .name }}
+              protocol: UDP
+          {{- end }}
+          {{- end }}
+          {{- if .tcp }}
+          {{- range .tcp.ports }}
+            - containerPort: {{ .port }}
+              name: {{ .name }}
+              protocol: TCP
+          {{- end }}
+          {{- end }}
+          {{- end }}
+          resources:
+{{ toYaml .Values.graylog.resources | indent 12 }}
+          livenessProbe:
```

Comment:
Update the following From`{ livenessProbe } To { readinessProbe }`

Reply:
@jmakanjuola there is `readinessProbe` few lines below.